pkg/util/signal/exit.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Copyright 2010 The Go Authors. All rights reserved.
```

Comment:
I don't think we need this license title in this file.  `pkg/util/signal/signal_posix.go` has this license title because it copied `getGoroutineStacks()` function from go source code.